style.css

```diff
@@ -0,0 +1,7 @@
+@import url('https://fonts.googleapis.com/css2?family=Caveat:wght@400;700&display=swap;family = Poiret + One & display = swap" rel = "таблица стилей">семья = Lobster &семья = Lora &Семейство = Open + Санс &семья = Pacifico &семья = Робото &');
+ * {
+    font-family: 'Caveat', cursive;
+  }
+h1,h2 {
+  font-family: 'Lora', cursive;
+}
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Unexpected missing end-of-source newline (no-missing-end-of-source-newline)](https://app.codacy.com/gh/WebHeroSchool/Sharingan/pullRequest?prid=5582230)